curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/-iterate-through-the-keys-of-an-object-with-a-for...in-statement.english.md

```diff
@@ -37,73 +53,76 @@ tests:
 <div id='js-seed'>
 
 ```js
-let users = {
+function countOnline(usersObj) {
+  // change code below this line
+
+  // change code above this line
+}
+```
+
+</div>
+
+### After Test
```

Comment:
```suggestion
### Before Test
```
See below.

Reply:
It is fine being After Test.  It does not change the outcome.